deployment/chainloop/README.md

```diff
@@ -532,6 +532,7 @@ chainloop config save \
 | `controlplane.referrerSharedIndex.allowedOrgs` | List of UUIDs of organizations that are allowed to publish to the shared index                  | `[]`                                               |
 | `controlplane.onboarding.name`                 | Name of the organization to onboard                                                             |                                                    |
 | `controlplane.onboarding.role`                 | Role of the organization to onboard                                                             |                                                    |
+| `controlplane.prometheus_integration.org_name` | Name of the organization to integrate with Prometheus                                           |                                                    |
```

Comment:
I usually leave deployment settings to another PR. I'd suggest to do the same here.

This feature is not available yet so we can not release a chart yet.